CONTRIBUTING.md

```diff
@@ -17,27 +17,53 @@ Cloudant-client is written in Java and uses maven as its build tool.
 
 Follow the instructions for your platform.
 
-### Maven
-
-Using brew
+### Gradle
 
-```bash
-$ brew install maven
-```
+The project uses the gradle wrapper to download  specified version of gradle.
```

Comment:
Useful to add the command line to run, or at least a link to using Gradle wrapper.